docs/using-mina/how-to-use-zkapp.mdx

```diff
@@ -3,12 +3,6 @@ title: How to Use a zkApp
 hide_title: true
 ---
 
-:::info
```

Comment:
The purpose of the callout has changed, so we can omit all of these banners except for the two pages as indicated